autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -15,6 +15,40 @@
     get_all_terms, get_recent_terms,
 )
 
+# Map from parsed instructional method to section.instructional_method choice
+_INSTRUCTIONAL_METHODS = {
+    # Normal instructional methods
+    'Traditional Face-to-Face (F2F)': Section.F2F,
+    'Internship': Section.INTERNSHIP,
+    'Non-traditional': Section.NONTRADITIONAL,
+    'Web Based': Section.WEB_BASED,
+    'Study Abroad': Section.STUDY_ABROAD,
+
+    # Coronavirus instructional methods
+    'Remote Only': Section.REMOTE,
+    'F2F with Remote Option': Section.F2F_REMOTE_OPTION,
+    'Mixed, F2F and Remote Meetings': Section.MIXED_F2F_REMOTE,
+}
+
+# Instructional methods to apply Section.web to
+_WEB_INSTRUCTIONAL_METHODS = set((
```

Comment:
Rename this to remote since that's what it's eventually going to be

On that note, should this be based on #446? Or maybe vice-versa?

Reply:
If you're fine with merging this after I address your comments, we might as well rebase #446 with master after this gets merged. I'd rather not rebase this branch since it adds functionality that we'll want on the site and don't want it to get held up by the other PR getting merged